src/components/CanvasText.vue

```diff
@@ -19,8 +19,8 @@ export default Vue.extend({
       context.clearRect(0, 0, canvasElement.width, canvasElement.height);
       context.fillStyle = "black";
       context.font = "20px Arial";
-      const textSize = context.measureText(binding.value);
-      canvasElement.setAttribute("width", textSize.width + 20 + "px");
+      const textWidth = context.measureText(binding.value).width || 450;
+      canvasElement.setAttribute("width", textWidth + 20 + "px");
```

Comment:
What is the magic 20?

Reply:
It compensates left padding from
 `context.fillText(binding.value, 15, 20);`

With padding
![image](https://user-images.githubusercontent.com/7165377/142145008-bc297a6c-8918-4c17-a5ed-0092b8d60fc6.png)

Without
![image](https://user-images.githubusercontent.com/7165377/142145048-0d9352ee-26aa-4dc4-8b24-7949b87dfb68.png)

Reply:
Maybe better to name that constant, even if we use it more than in on place?

Reply:
Thanks a good idea! Thanks